test-spec.yml

```diff
@@ -1,9 +1,26 @@
 sequence:
   - tests:
-      - name: ltp
+      - name: kernel version
         command:
-          - /usr/share/fio-tests/ltp.sh
-    # repeat not specified, so runs one time
+          - if [ $(uname -r) = "5.10.9-lmp-standard" ]; then /bin/true; else /bin/false; fi
```

Comment:
how about we just verify "lmp-standard" is in the output or something? otherwise we'll have to change this for every LMP update and users that copy this will see failed tests?

Reply:
That's why I asked about parameters. Each build might have a different kernel version. IMHO this should be set from the result of CI build. I can remove it for now.

Reply:
the problem is fiotest has no context to set this parameter. its just triggered with a 'run tests' which is triggered by aktualizr which found a new Target. Dynamic things like this need to be able to find a source of truth. For example, this test would need to look at `https://api.foundries.io/projects/lmp/builds/1180/runs/intel-corei7-64/` and somehow determine the kernel from a build artifact.